section4/exercises/classMethods.js

```diff
@@ -28,13 +28,32 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
-
   // ADD CODE
+  changeProtein() {
+    this.protein = "Duck";
```

Comment:
This method changes the protein to the same thing every time... how can you use a parameter to make this method dynamic?